cluster/cluster.go

```diff
@@ -374,11 +376,11 @@ func parseAuditLogConfig(clusterFile string, rkeConfig *v3.RancherKubernetesEngi
 	if services["kube-api"] == nil {
 		return nil
 	}
-	kubeapi := services["kube-api"].(map[string]interface{})
-	if kubeapi["audit_log"] == nil {
+	kubeAPI := services["kube-api"].(map[string]interface{})
```

Comment:
Not sure how the team feels about this, but since Go 1.18, a lot of people just use `any` instead of `interface{}`. Since the PR also sets the minimum Go version to 1.19 it should be safe to change it

Reply:
thank you for the advice 😄 
I will keep using the old fashion this time and leave the refactoring for another PR.

Reply:
Sure, that's fine -- thank you!